plonky2/src/plonk/circuit_builder.rs

```diff
@@ -1023,15 +1026,18 @@ impl<F: RichField + Extendable<D>, const D: usize> CircuitBuilder<F, D> {
         let max_fft_points = 1 << (degree_bits + max(rate_bits, log2_ceil(quotient_degree_factor)));
         let fft_root_table = fft_root_table(max_fft_points);
 
-        let constants_sigmas_vecs = [constant_vecs, sigma_vecs.clone()].concat();
-        let constants_sigmas_commitment = PolynomialBatch::<F, C, D>::from_values(
-            constants_sigmas_vecs,
-            rate_bits,
-            PlonkOracle::CONSTANTS_SIGMAS.blinding,
-            cap_height,
-            &mut timing,
-            Some(&fft_root_table),
-        );
+        let mut constants_sigmas_commitment = PolynomialBatch::<F, C, D>::default();
```

Comment:
nit: I think
`let constants_sigmas_commitment = if commit_to_sigma {
    ...
} else {
    PolynomialBatch::<F, C, D>::default()
}` would be cleaner